setup.py

```diff
@@ -40,7 +40,7 @@
                       'boto >= 2.0b4',
                       'bunch >= 1.0.0',
                       'configobj',
-                      'six',
+                      'six >= 1.9', # python-openstackclient won't work properly with less
```

Comment:
This was recently changed in 9d59d6bd93de47664c939fea331ced6593e8ae16

I believe because of an issue with mock. Would this change reintroduce that?